src/main/java/HelloWorld.java

```diff
@@ -0,0 +1,12 @@
+/**
+ *
+ * @author tkardozo
+ */
+public class HelloWorld {
+
+    public static void main(String[] args) {
+        //AHAHAHAH
+	System.out.print("GoNutri FTW");
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Replace this use of System.out or System.err by a logger. ([squid:S106](https://sonarcloud.io/organizations/bdomingues-github/rules#rule_key=squid%3AS106))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=com.mycompany%3AhelloTQS&pullRequest=1&issues=AWM8E7oLneya_4cH99ho&open=AWM8E7oLneya_4cH99ho)

[ISSUE_KEY]: AWM8E7oLneya_4cH99ho